### DISCUSSION TITLE
what transaction error does this mean?

### DISCUSSION THREAD

**Discussion by a-rya13** [Cyfrin/foundry-full-course-cu#2523] [Q&A]
![Screenshot 2024-09-21 225357](https://github.com/user-attachments/assets/2432a77c-0b25-4946-a4e1-f9c898153ef0)
I have funds in my account sepolia test network and that too 0.097SepoliaEth. so what should i do now?

**Comment by EngrPips** [comment]
Hello @a-rya13, are you sure you are using the private associated with the wallet you fund?

**Reply by a-rya13** [reply]
@EngrPips, yes I am sure..I have checked it many times

**Comment by EngrPips** [comment]
> ![Screenshot 2024-09-21 225357](https://private-user-images.githubusercontent.com/137289602/369637759-2432a77c-0b25-4946-a4e1-f9c898153ef0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcwMTI5OTAsIm5iZiI6MTcyNzAxMjY5MCwicGF0aCI6Ii8xMzcyODk2MDIvMzY5NjM3NzU5LTI0MzJhNzdjLTBiMjUtNDk0Ni1hNGUxLWY5Yzg5ODE1M2VmMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMlQxMzQ0NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZDE3MmU3NjRlZGJmY2E0ODMxZDk4YTUxNTU4NTJkNmU0MDk1NjQ4MjNhYzU5NDg2NmM1MzdjMzRjMzAwNDZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LwgUJikePiGtfCu6BhQOj1yZ65WamogwCk_S-Mv3KJ4) I have funds in my account sepolia test network and that too 0.097SepoliaEth. so what should i do now?

On what chain did you run this process?

**Reply by a-rya13** [reply]
https://eth-sepolia.g.alchemy.com/v2/xlhu4idqNlzJOqb964yDWZD8rcHOzs8v

**Reply by EngrPips** [reply]
I will say try another `private key` that has funds in it, and let's see. I am unsure what is wrong. You claimed that you are using the private key associated with a wallet that has funds in it, and it should work if that is the case

**Reply by a-rya13** [reply]
sure sir

**Reply by EngrPips** [reply]
Interesting
